python/ray/train/lightning/__init__.py

```diff
@@ -23,6 +23,16 @@
     RayTrainReportCallback,
 )
 
+from packaging.version import Version
```

Comment:
Can you move this whole block to be immediately after importing `pytorch_lightning` at the top of the file?

Reply:
oh it will raise a lint error, saying all imports should be at the top of the whole file